distribution/docs/src/main/resources/_contents/introductioncontents.adoc

```diff
@@ -0,0 +1,44 @@
+Distributed Data Framework {branding} is a free and open-source common data layer that abstracts services and business logic from the underlying data structures to enable rapid integration of new data sources.
```

Comment:
I know I'm being picky here, but could we have a replacement for the expanded version, say `branding-expanded`, and change this to be `{branding-expanded} ({branding})`? Or just leave the acronym expansion out?